components/compositing/webview_renderer.rs

```diff
@@ -88,7 +88,7 @@ pub(crate) struct WebViewRenderer {
     touch_handler: TouchHandler,
     /// "Desktop-style" zoom that resizes the viewport to fit the window.
     pub page_zoom: Scale<f32, CSSPixel, DeviceIndependentPixel>,
-    /// "Mobile-style" zoom that does not reflow the page.
+    /// "Mobile-style" zoom that does not reflow the page i.e pinch zoom or viewport meta initial scale zoom.
```

Comment:
```suggestion
    /// "Mobile-style" zoom that does not reflow the page, i.e. pinch zoom.
```
It's not always equal to the initial scale zoom